### PR TITLE
Add Turkish translatiın

### PR DESCRIPTION
**Pull request by berkaygunduzz**
### Changes
Followed [instructions](https://github.com/ashutoshgngwr/noice/blob/master/docs/CONTRIBUTING.md#translations) for Turkish translate

### CODE REVIEW

**Comment by berkaygunduzz** [comment]
@ashutoshgngwr Why it's failed?

**Comment by ashutoshgngwr** [comment]
@berkaygunduzz Still running UI tests. Should pass after that.

**Comment by berkaygunduzz** [comment]
Then ok, waiting.

File: fastlane/metadata/android/tr/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Pek çok insan için, ufak bir arka plan gürültüsü sakinleşmeye ve odaklanmaya
+yardımcı oluyor. Bazı durumlarda üretkenliği bile arttrıyor. Ayrıca arka plan
+sesi (ambiyans) stresin etkilerini azaltmaya da yardımcı olabilir.
+
+Noice çevresel sesleri kullanarak kendi ambiyans kayıtlarınızı oluşturmanızı
+sağlayan bir uygulamadır. İşinize odaklanmak ya da sakin, barış dolu bir ortam yaratarak uyumanızı sağlamak adına rahatsız edecek sesleri baskılayacak mükemmel kayıtlarınızı oluşturun.
```
Comment (ashutoshgngwr) [inline fastlane/metadata/android/tr/full_description.txt:6]: New lines will start new paragraphs. Please remove them to keep paragraphs as single lines.

```suggestion
Pek çok insan için, ufak bir arka plan gürültüsü sakinleşmeye ve odaklanmaya yardımcı oluyor. Bazı durumlarda üretkenliği bile arttrıyor. Ayrıca arka plan sesi (ambiyans) stresin etkilerini azaltmaya da yardımcı olabilir.

Noice çevresel sesleri kullanarak kendi ambiyans kayıtlarınızı oluşturmanızı sağlayan bir uygulamadır. İşinize odaklanmak ya da sakin, barış dolu bir ortam yaratarak uyumanızı sağlamak adına rahatsız edecek sesleri baskılayacak mükemmel kayıtlarınızı oluşturun.
```

**Review by ashutoshgngwr — CHANGES_REQUESTED** [review]
Needs a few minor changes. Everything else looks great. Nice work!

File: app/src/main/res/values-tr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    Kaydettiğiniz bir karışımı yeniden kullanmak için gezinti çubuğundan Kayıtlar
+    listesini ziyaret edebilirsiniz.
+  </string>
+  <string name="appintro__timer_title">Uyumakta veya uyanmakta zorluk mu çekiyorsunuz?</string>
+  <string name="appintro__timer_desc">
+    Ufak bir ambiyans sizin uyumanızda zahmetsizce yardımcı olabilir. Uyku
+    Zamanlayıcı kullanarak Noice\'in sesi kendiliğinden kesmesini sağlayabilirsiniz.
+    Benzer şekilde sakin bir ortamda uyanmak için Uyandırma Zamanlayıcı kullanarak
+    alarmı ayarlayabilirsiniz
+  </string>
```
[the remark this answers is not in the document]
Comment (berkaygunduzz) [reply app/src/main/res/values-tr/strings.xml:149]: I will try, just give me a little time

[the remark this answers is not in the document]
Comment (berkaygunduzz) [reply app/src/main/res/values-tr/strings.xml]: Now, it's not word by word translation, but gives similar meaning.
For example now title says "Are you having problems about sleep?"

[the remark this answers is not in the document]
Comment (berkaygunduzz) [reply app/src/main/res/values-tr/strings.xml]: Where is the problem?

[the remark this answers is not in the document]
Comment (berkaygunduzz) [reply app/src/main/res/values-tr/strings.xml]: Why it gives me [an error](https://github.com/ashutoshgngwr/noice/pull/428/commits/7511b9939a5077a69f9898411f0de83cbbabddeb) while one line was too long? I fixed it but what's about?

**Review by ashutoshgngwr — APPROVED** [review]
LGTM :+1:

**Pull request merged**